lib/todolist_web/templates/task/task.html.eex

```diff
@@ -0,0 +1,11 @@
+<section class="row">
+  <article class="column">
+    <h2>Tasks</h2>
+    <ul>
```

Comment:
Probably you meant:

```eex
<ul>
  <%= for task <- @tasks do %>
    <li> <%= task %> </li>
  <% end %>
</ul>
```

Your code is creating N `<p>`-tags in 1 html list tag

Reply:
Done